shell/platform/common/cpp/client_wrapper/plugin_registrar.cc

```diff
@@ -87,6 +93,26 @@ void BinaryMessengerImpl::Send(const std::string& channel,
                               message_size);
 }
 
+void BinaryMessengerImpl::Send(const std::string& channel,
+                               const uint8_t* message,
+                               const size_t message_size,
+                               BinaryReply reply) const {
+  struct Captures {
+    BinaryReply reply;
+  };
+  auto captures = new Captures();
```

Comment:
This will leak if `FlutterDesktopMessengerSendWithReply` fails (the early return codepath). We should make `...SendWithReply` return a boolean for whether sending succeeded, so that if it returns false functions like this one can do cleanup.

Reply:
Done.

Reply:
Done